<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>690</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1217</width>
    <height>831</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/download-icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/view-refresh-4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1181</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>511</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/VISITEURBUUT.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>511</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/bUTT evenement.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1181</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Gestion Visiteur</string>
      </property>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>390</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/HOMEBUUT.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>1081</width>
         <height>451</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Ajouter Visiteur</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>100</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prénom :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>210</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>150</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>160</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Email :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>220</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tél :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>270</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Age :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>20</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>270</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>320</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>HOMME</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>320</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>FEMME</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>330</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Sexe :</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_16">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>360</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter </string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1181</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Gestion Evènement</string>
      </property>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>390</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/HOMEBUUT.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/home-icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/dossier.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="action2"/>
    <addaction name="separator"/>
    <addaction name="action3"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelpe">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelpe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
